tests/FakeItEasy.Analyzer.Tests.Shared/FakeItEasy.Analyzer.Tests.Shared.projitems

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>d74665d8-d046-4c38-81d6-8aa7ead68468</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>FakeItEasy.Analyzer.Tests.Shared</Import_RootNamespace>

```

Comment:
Similar comment here. Naively, I'd say "Shared" should be dropped from the default namespace.